scripts/setupTests.js

```diff
@@ -91,5 +92,7 @@ expect.extend({
   }
 });
 
+configure({asyncUtilTimeout: 400});
```

Comment:
https://github.com/testing-library/dom-testing-library/issues/524#issuecomment-1206509128, adding this seemed to help with the build errors a bit

Reply:
does this still happen now after devon's work?

Reply:
Not that I can tell, I had kept it in since it felt like a good guard against a potential hard to debug issue but I'll remove it for now